pkg/labels/labels.go

```diff
@@ -66,19 +67,16 @@ func (ls Labels) String() string {
 // It uses an byte invalid character as a separator and so should not be used for printing.
 func (ls Labels) Bytes(buf []byte) []byte {
 	b := bytes.NewBuffer(buf[:0])
-	b.WriteByte('{')
+	b.WriteByte(labelSep)
 	for i, l := range ls {
 		if i > 0 {
-			b.WriteByte(',')
-			b.WriteByte(' ')
+			b.WriteByte(sep)
 		}
 		b.WriteString(l.Name)
-		b.WriteByte('=')
 		b.WriteByte(sep)
 		b.WriteString(l.Value)
-		b.WriteByte(sep)
 	}
-	b.WriteByte('}')
+	b.WriteByte(labelSep)
```

Comment:
I think we don't need this trailing byte. Just have the leading byte should be enough as that marks the beginning of a new label set in case of concatenation.